torchx/test/torchx/nx_test.exs

```diff
@@ -62,6 +62,25 @@ defmodule Torchx.NxTest do
 
         test_binary_op(op, type_a, type_b)
       end
+
+      test "#{op}(#{Nx.Type.to_string(type_a)}, #{Nx.Type.to_string(type_b)}) broadcast" do
```

Comment:
not sure if we should add all of these, though, because this adds 1k tests to the file (praise the power of permutations)

Reply:
Probably too many. Perhaps we should get a few as reference only?

Reply:
I've changed the generator to only test for same-type broadcasts instead (1k -> 100 tests instead)